packages/console/src/components/Transfer/index.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from 'classnames';
+import React, { ReactNode, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import CircleMinus from '@/icons/CircleMinus';
+import CirclePlus from '@/icons/CirclePlus';
+import Dragable from '@/icons/Dragable';
```

Comment:
Still... Drag`g`able. Please search and replace globally, including file names

Reply:
updated, sorry for the typo issues.